src/caches/beliefpropagationcache.jl

```diff
@@ -0,0 +1,240 @@
+default_initial_message(inds_e) = ITensor[denseblocks(delta(inds_e))]
```

Comment:
This would overwrite `default_message(bp_cache) = bp_cache.default_message`t. In principle it should be called `default_default_message` I guess but that is a bit confusint

Reply:
Can't you restrict the other version to `default_message(bp_cache::BeliefPropagationCache)`?

Reply:
Good point yes I forgot to make that restriction